library/advanced-search.en.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: Advanced Search
+Category: Library
+Template: document
+Tags: support
+Date: 2017-07-20
+Slug: advanced-search
+Summary:
+Lang: en
+toc_run: true
```

Comment:
If you enable this option, you don't need the anchor links (< a name ...>), they will be added automatically to each heading.